### PR TITLE
Feature/highstockchart2

### PR DESCRIPTION
**Pull request by hawkish**
This PR closes # (reference issue number here)

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] cookbook application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?
Highstock has been added.
## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: apps/cookbook-e2e/src/plugins/index.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 // ***********************************************************
-// This example plugins/index.js can be used to load plugins
+// This example plugins/index.ts can be used to load plugins
```
Comment (jakobe) [inline apps/cookbook-e2e/src/plugins/index.js]: Revert 🔙

File: apps/cookbook-e2e/src/support/index.ts [hunk 2]
```diff
@@ -1,5 +1,5 @@
 // ***********************************************************
-// This example support/index.js is processed and
+// This example support/index.ts is processed and
```
Comment (jakobe) [inline apps/cookbook-e2e/src/support/index.ts]: Revert 🔙

File: apps/cookbook/src/app/examples/examples.routes.ts [hunk 3]
```diff
@@ -217,6 +218,10 @@ export const routes: Routes = [
     path: 'chart',
     component: ChartExampleComponent,
   },
+  {
+    path: 'highstock-chart',
```
Comment (jakobe) [inline apps/cookbook/src/app/examples/examples.routes.ts:222]: Consider naming:
```suggestion
    path: 'chart-stock',
```

File: apps/cookbook/src/app/examples/highstock-chart-example/highstock-chart-example.component.html [hunk 4]
```diff
@@ -0,0 +1,10 @@
+<kirby-card>
+  <kirby-card-header [title]="'Highstock Chart'"></kirby-card-header>
```
Comment (jakobe) [inline apps/cookbook/src/app/examples/highstock-chart-example/highstock-chart-example.component.html:2]: Consider naming:
```suggestion
  <kirby-card-header [title]="'Stock Chart'"></kirby-card-header>
```

File: apps/cookbook/src/app/examples/highstock-chart-example/highstock-chart-example.component.ts [hunk 5]
```diff
@@ -0,0 +1,296 @@
...
+
+import { Options } from 'highcharts';
+
+import {
+  HighstockDataPoint,
+  stockChartOptions,
+} from '@kirbydesign/designsystem/components/highstock-chart/options/highstock-chart-options';
+
+@Component({
+  selector: 'cookbook-highstock-chart-example',
```
Comment (jakobe) [inline apps/cookbook/src/app/examples/highstock-chart-example/highstock-chart-example.component.ts:11]: Consider naming:
```suggestion
  selector: 'cookbook-chart-example-stock',
```

File: apps/cookbook/src/app/examples/highstock-chart-example/highstock-chart-example.component.ts [hunk 6]
```diff
@@ -0,0 +1,296 @@
+import { Component, Inject, LOCALE_ID, OnInit } from '@angular/core';
+
+import { Options } from 'highcharts';
+
+import {
+  HighstockDataPoint,
+  stockChartOptions,
+} from '@kirbydesign/designsystem/components/highstock-chart/options/highstock-chart-options';
```
Comment (jakobe) [inline apps/cookbook/src/app/examples/highstock-chart-example/highstock-chart-example.component.ts:8]: Ensure is exported from root:
```suggestion
} from '@kirbydesign/designsystem';
```

File: apps/cookbook/src/app/examples/highstock-chart-example/highstock-chart-example.component.ts [hunk 7]
```diff
@@ -0,0 +1,296 @@
+import { Component, Inject, LOCALE_ID, OnInit } from '@angular/core';
+
+import { Options } from 'highcharts';
+
+import {
+  HighstockDataPoint,
```
Comment (jakobe) [inline apps/cookbook/src/app/examples/highstock-chart-example/highstock-chart-example.component.ts:6]: Consider naming:
```suggestion
  stockChartDataPoint,
```

File: apps/cookbook/src/app/examples/highstock-chart-example/highstock-chart-example.component.ts [hunk 8]
```diff
@@ -0,0 +1,296 @@
...
+  stockChartOptions,
+} from '@kirbydesign/designsystem/components/highstock-chart/options/highstock-chart-options';
+
+@Component({
+  selector: 'cookbook-highstock-chart-example',
+  templateUrl: './highstock-chart-example.component.html',
```
Comment (jakobe) [inline apps/cookbook/src/app/examples/highstock-chart-example/highstock-chart-example.component.ts:12]: Consider naming:
```suggestion
  templateUrl: './chart-example-stock.component.html',
```

File: apps/cookbook/src/app/examples/highstock-chart-example/highstock-chart-example.component.ts [hunk 9]
```diff
@@ -0,0 +1,296 @@
...
+  styleUrls: ['./highstock-chart-example.component.scss'],
```
Comment (jakobe) [inline apps/cookbook/src/app/examples/highstock-chart-example/highstock-chart-example.component.ts:13]: Consider naming:
```suggestion
  styleUrls: ['./chart-example-stock.component.scss'],
```

File: apps/cookbook/src/app/examples/highstock-chart-example/highstock-chart-example.component.ts [hunk 10]
```diff
@@ -0,0 +1,296 @@
...
+})
+export class HighstockChartExampleComponent implements OnInit {
```
Comment (jakobe) [inline apps/cookbook/src/app/examples/highstock-chart-example/highstock-chart-example.component.ts:15]: Consider naming:
```suggestion
export class ChartExampleStockComponent implements OnInit {
```

File: apps/cookbook/src/app/showcase/showcase.common.ts [hunk 11]
```diff
@@ -36,6 +36,7 @@ import { ItemShowcaseComponent } from '../showcase/item-showcase/item-showcase.c
 import { DividerShowcaseComponent } from '../showcase/divider-showcase/divider-showcase.component';
 import { DropdownShowcaseComponent } from '~/app/showcase/dropdown-showcase/dropdown-showcase.component';
 import { ReorderListShowcaseComponent } from '../showcase/reorder-list-showcase/reorder-list-showcase.component';
+import { HighstockChartShowcaseComponent } from '~/app/showcase/highstock-chart-showcase/highstock-chart-showcase.component';
```
Comment (jakobe) [inline apps/cookbook/src/app/showcase/showcase.common.ts:39]: Use relative import
```suggestion
import { HighstockChartShowcaseComponent } from './highstock-chart-showcase/highstock-chart-showcase.component';
```

File: apps/cookbook/src/app/showcase/showcase.routes.ts [hunk 12]
```diff
@@ -36,6 +36,7 @@ import { ItemShowcaseComponent } from '../showcase/item-showcase/item-showcase.c
 import { DividerShowcaseComponent } from './divider-showcase/divider-showcase.component';
 import { DropdownShowcaseComponent } from '~/app/showcase/dropdown-showcase/dropdown-showcase.component';
 import { ReorderListShowcaseComponent } from './reorder-list-showcase/reorder-list-showcase.component';
+import { HighstockChartShowcaseComponent } from '~/app/showcase/highstock-chart-showcase/highstock-chart-showcase.component';
```
Comment (jakobe) [inline apps/cookbook/src/app/showcase/showcase.routes.ts:39]: Use relative import:
```suggestion
import { HighstockChartShowcaseComponent } from './highstock-chart-showcase/highstock-chart-showcase.component';
```

File: apps/cookbook/src/app/showcase/showcase.routes.ts [hunk 13]
```diff
@@ -111,6 +112,10 @@ export const routes: Routes = [
         path: 'chart',
         component: ChartShowcaseComponent,
       },
+      {
+        path: 'highstock-chart',
```
Comment (jakobe) [inline apps/cookbook/src/app/showcase/showcase.routes.ts:116]: Consider naming:
```suggestion
        path: 'chart-stock',
```

File: libs/designsystem/src/lib/components/highstock-chart/highstock-chart.component.html [hunk 14]
```diff
@@ -0,0 +1 @@
+<div #chartContainer [style.height]="height + 'px'"></div>
```
Comment (jakobe) [inline libs/designsystem/src/lib/components/highstock-chart/highstock-chart.component.html:1]: See https://github.com/kirbydesign/designsystem/blob/master/libs/designsystem/src/lib/components/chart/chart.component.ts on how to get rid of redundant div

File: libs/designsystem/src/lib/components/highstock-chart/highstock-chart.component.ts [hunk 15]
```diff
@@ -0,0 +1,88 @@
...
+
+import { annotations, HighstockDataPoint } from './options/highstock-chart-options';
+
+// @ts-ignore
+AnnotationsModule(Highcharts);
+
+@Component({
+  selector: 'kirby-highstock-chart',
+  templateUrl: './highstock-chart.component.html',
+  styleUrls: ['./highstock-chart.component.scss'],
```
Comment (jakobe) [inline libs/designsystem/src/lib/components/highstock-chart/highstock-chart.component.ts:15]: Delete if empty...?

File: libs/designsystem/src/lib/components/highstock-chart/highstock-chart.component.ts [hunk 16]
```diff
@@ -0,0 +1,88 @@
...
+})
+export class HighstockChartComponent {
```
Comment (jakobe) [inline libs/designsystem/src/lib/components/highstock-chart/highstock-chart.component.ts:17]: Consider naming:
```suggestion
export class StockChartComponent {
```

File: libs/designsystem/src/lib/components/highstock-chart/highstock-chart.component.ts [hunk 17]
```diff
@@ -0,0 +1,88 @@
+import { Component, ElementRef, Inject, Input, LOCALE_ID, ViewChild } from '@angular/core';
+
+import { Options } from 'highcharts';
+import * as Highcharts from 'highcharts/highstock';
```
Comment (jakobe) [inline libs/designsystem/src/lib/components/highstock-chart/highstock-chart.component.ts:4]: Consider only importing what's used:
```suggestion
import { stockChart } from 'highcharts/highstock';
```

File: libs/designsystem/src/lib/components/highstock-chart/highstock-chart.component.ts [hunk 18]
```diff
@@ -0,0 +1,88 @@
...
+
+  @ViewChild('chartContainer', { static: true }) chartContainer: ElementRef;
+
+  chart: Highcharts.Chart;
+
+  constructor(@Inject(LOCALE_ID) private locale: string) {}
+
+  onOptionsChanges(options: Options) {
+    this._options = options;
+    this.chart = Highcharts.stockChart(this.chartContainer.nativeElement, this._options);
```
Comment (jakobe) [inline libs/designsystem/src/lib/components/highstock-chart/highstock-chart.component.ts:53]: Consider only importing what's used:
```suggestion
    this.chart = stockChart(this.chartContainer.nativeElement, this._options);
```

File: libs/designsystem/src/lib/components/highstock-chart/options/highstock-chart-options.ts [hunk 19]
```diff
@@ -0,0 +1,221 @@
+import { formatNumber } from '@angular/common';
+
+import { Options } from 'highcharts';
+import * as Highcharts from 'highcharts/highstock';
+
+import { ColorHelper } from '../../../helpers';
+
+export interface HighstockDataPoint {
```
Comment (jakobe) [inline libs/designsystem/src/lib/components/highstock-chart/options/highstock-chart-options.ts:8]: Consider naming:
```suggestion
export interface StockChartDataPoint {
```

File: libs/designsystem/src/lib/components/highstock-chart/options/highstock-chart-options.ts [hunk 20]
```diff
@@ -0,0 +1,221 @@
...
+  x: number;
+  y: number;
+  id: 'min' | 'max' | '';
+}
+
+export function stockChartOptions(locale: string, height: number) {
```
Comment (jakobe) [inline libs/designsystem/src/lib/components/highstock-chart/options/highstock-chart-options.ts:14]: Align with existing chart options - export as const - see https://github.com/kirbydesign/designsystem/blob/master/libs/designsystem/src/lib/components/chart/options/bar.ts

File: libs/designsystem/src/lib/components/highstock-chart/options/highstock-chart-options.ts [hunk 21]
```diff
@@ -0,0 +1,221 @@
+import { formatNumber } from '@angular/common';
+
+import { Options } from 'highcharts';
+import * as Highcharts from 'highcharts/highstock';
```
Comment (jakobe) [inline libs/designsystem/src/lib/components/highstock-chart/options/highstock-chart-options.ts:4]: Consider only importing what's used:
```suggestion
import { dateFormat } from 'highcharts/highstock';
```

File: libs/designsystem/src/lib/kirby.module.ts [hunk 22]
```diff
@@ -115,6 +116,7 @@ const exportedDeclarations = [
   DropdownComponent,
   InfiniteScrollDirective,
   LoadingOverlayComponent,
+  HighstockChartComponent,
```
Comment (jakobe) [inline libs/designsystem/src/lib/kirby.module.ts:119]: Consider moving to after ChartComponent for readability... 👀

File: libs/designsystem/src/lib/kirby.module.ts [hunk 23]
```diff
@@ -71,6 +71,7 @@ import { BreakpointHelperService } from './components/grid/breakpoint-helper.ser
 import { LoadingOverlayService } from './components/loading-overlay/fullscreen-loading-overlay/loading-overlay.service';
 import { ResizeObserverFactory } from './components/shared/resize-observer/resize-observer.factory';
 import { ResizeObserverService } from './components/shared/resize-observer/resize-observer.service';
+import { HighstockChartComponent } from './components/highstock-chart/highstock-chart.component';
```
Comment (jakobe) [inline libs/designsystem/src/lib/kirby.module.ts:74]: Consider moving to after ChartComponent for readability... 👀

File: apps/cookbook-e2e/src/plugins/index.js (hunk 1 above)
Comment (hawkish) [reply apps/cookbook-e2e/src/plugins/index.js]: Done

File: apps/cookbook-e2e/src/support/index.ts (hunk 2 above)
Comment (hawkish) [reply apps/cookbook-e2e/src/support/index.ts]: Done.

File: apps/cookbook/src/app/examples/examples.routes.ts (hunk 3 above)
Comment (hawkish) [reply apps/cookbook/src/app/examples/examples.routes.ts:222]: Done.

File: apps/cookbook/src/app/examples/highstock-chart-example/highstock-chart-example.component.html (hunk 4 above)
Comment (hawkish) [reply apps/cookbook/src/app/examples/highstock-chart-example/highstock-chart-example.component.html:2]: Done.

File: apps/cookbook/src/app/examples/highstock-chart-example/highstock-chart-example.component.ts (hunk 5 above)
Comment (hawkish) [reply apps/cookbook/src/app/examples/highstock-chart-example/highstock-chart-example.component.ts:11]: Done.

File: libs/designsystem/src/lib/components/highstock-chart/options/highstock-chart-options.ts (hunk 19 above)
Comment (hawkish) [reply libs/designsystem/src/lib/components/highstock-chart/options/highstock-chart-options.ts:8]: Done.

File: libs/designsystem/src/lib/components/highstock-chart/options/highstock-chart-options.ts (hunk 20 above)
Comment (hawkish) [reply libs/designsystem/src/lib/components/highstock-chart/options/highstock-chart-options.ts:14]: Done. It confused Anders as a const, but I'll revert it as such.

File: libs/designsystem/src/lib/components/highstock-chart/options/highstock-chart-options.ts (hunk 21 above)
Comment (hawkish) [reply libs/designsystem/src/lib/components/highstock-chart/options/highstock-chart-options.ts:4]: Done.

File: libs/designsystem/src/lib/kirby.module.ts (hunk 22 above)
Comment (hawkish) [reply libs/designsystem/src/lib/kirby.module.ts:119]: Done.

File: libs/designsystem/src/lib/kirby.module.ts (hunk 23 above)
Comment (hawkish) [reply libs/designsystem/src/lib/kirby.module.ts:74]: Done

File: apps/cookbook/src/app/examples/highstock-chart-example/highstock-chart-example.component.ts (hunk 6 above)
Comment (hawkish) [reply apps/cookbook/src/app/examples/highstock-chart-example/highstock-chart-example.component.ts:8]: Done.

File: apps/cookbook/src/app/examples/highstock-chart-example/highstock-chart-example.component.ts (hunk 7 above)
Comment (hawkish) [reply apps/cookbook/src/app/examples/highstock-chart-example/highstock-chart-example.component.ts:6]: Done.

File: apps/cookbook/src/app/examples/highstock-chart-example/highstock-chart-example.component.ts (hunk 8 above)
Comment (hawkish) [reply apps/cookbook/src/app/examples/highstock-chart-example/highstock-chart-example.component.ts:12]: Done

File: apps/cookbook/src/app/examples/highstock-chart-example/highstock-chart-example.component.ts (hunk 9 above)
Comment (hawkish) [reply apps/cookbook/src/app/examples/highstock-chart-example/highstock-chart-example.component.ts:13]: Done.

File: apps/cookbook/src/app/examples/highstock-chart-example/highstock-chart-example.component.ts (hunk 10 above)
Comment (hawkish) [reply apps/cookbook/src/app/examples/highstock-chart-example/highstock-chart-example.component.ts:15]: Done.

File: apps/cookbook/src/app/showcase/showcase.common.ts (hunk 11 above)
Comment (hawkish) [reply apps/cookbook/src/app/showcase/showcase.common.ts:39]: Done.

File: apps/cookbook/src/app/showcase/showcase.routes.ts (hunk 12 above)
Comment (hawkish) [reply apps/cookbook/src/app/showcase/showcase.routes.ts:39]: Done.

File: apps/cookbook/src/app/showcase/showcase.routes.ts (hunk 13 above)
Comment (hawkish) [reply apps/cookbook/src/app/showcase/showcase.routes.ts:116]: Done.

File: libs/designsystem/src/lib/components/highstock-chart/options/highstock-chart-options.ts (hunk 20 above)
Comment (hawkish) [reply libs/designsystem/src/lib/components/highstock-chart/options/highstock-chart-options.ts:14]: Done.

File: libs/designsystem/src/lib/components/highstock-chart/options/highstock-chart-options.ts (hunk 21 above)
Comment (hawkish) [reply libs/designsystem/src/lib/components/highstock-chart/options/highstock-chart-options.ts:4]: Done.

File: libs/designsystem/src/lib/kirby.module.ts (hunk 22 above)
Comment (hawkish) [reply libs/designsystem/src/lib/kirby.module.ts:119]: Done

File: libs/designsystem/src/lib/kirby.module.ts (hunk 23 above)
Comment (hawkish) [reply libs/designsystem/src/lib/kirby.module.ts:74]: Done.

File: libs/designsystem/src/lib/components/highstock-chart/highstock-chart.component.ts (hunk 16 above)
Comment (hawkish) [reply libs/designsystem/src/lib/components/highstock-chart/highstock-chart.component.ts:17]: Done

File: libs/designsystem/src/lib/components/highstock-chart/highstock-chart.component.ts (hunk 17 above)
Comment (hawkish) [reply libs/designsystem/src/lib/components/highstock-chart/highstock-chart.component.ts:4]: Done.

File: libs/designsystem/src/lib/components/highstock-chart/highstock-chart.component.ts (hunk 18 above)
Comment (hawkish) [reply libs/designsystem/src/lib/components/highstock-chart/highstock-chart.component.ts:53]: Done.

File: libs/designsystem/src/lib/components/highstock-chart/highstock-chart.component.ts (hunk 15 above)
Comment (hawkish) [reply libs/designsystem/src/lib/components/highstock-chart/highstock-chart.component.ts:15]: Done.

File: libs/designsystem/src/lib/components/highstock-chart/highstock-chart.component.html (hunk 14 above)
Comment (hawkish) [reply libs/designsystem/src/lib/components/highstock-chart/highstock-chart.component.html:1]: Done.

File: apps/cookbook/src/app/examples/chart-example-stock/chart-example-stock.component.html [hunk 24]
```diff
@@ -1,10 +1,10 @@
 <kirby-card>
-  <kirby-card-header [title]="'Highstock Chart'"></kirby-card-header>
-  <kirby-highstock-chart
+  <kirby-card-header [title]="'Stock Chart'"></kirby-card-header>
```
Comment (jakobe) [inline apps/cookbook/src/app/examples/chart-example-stock/chart-example-stock.component.html]: Consider replacing with [One-time string initialization](https://angular.io/guide/template-syntax#one-time-string-initialization)
```suggestion
  <kirby-card-header title="Stock Chart"></kirby-card-header>
```

File: apps/cookbook/src/app/examples/chart-example-stock/chart-example-stock.component.html [hunk 25]
```diff
@@ -1,10 +1,10 @@
...
+  <kirby-chart-stock
     [description]="'Accessibility description goes here'"
```
Comment (jakobe) [inline apps/cookbook/src/app/examples/chart-example-stock/chart-example-stock.component.html]: Consider replacing with [One-time string initialization](https://angular.io/guide/template-syntax#one-time-string-initialization)
```suggestion
    description="Accessibility description goes here"
```

File: apps/cookbook/src/app/examples/chart-example-stock/chart-example-stock.component.html (hunk 24 above)
Comment (hawkish) [reply apps/cookbook/src/app/examples/chart-example-stock/chart-example-stock.component.html]: Done

File: apps/cookbook/src/app/examples/chart-example-stock/chart-example-stock.component.html (hunk 25 above)
Comment (hawkish) [reply apps/cookbook/src/app/examples/chart-example-stock/chart-example-stock.component.html]: Done.

File: apps/cookbook/src/app/examples/stock-chart-example/stock-chart-example.component.ts [hunk 26]
```diff
@@ -4,12 +4,22 @@ import { Options } from 'highcharts';
...
-})
+  template: `
+    <kirby-stock-chart
+      description="Accessibility description goes here"
+      [options]="options"
+      [data]="data"
+      [height]="height"
+    >
+    </kirby-stock-chart>
+  `,
```
Comment (jakobe) [inline apps/cookbook/src/app/examples/stock-chart-example/stock-chart-example.component.ts:17]: Whitespace is significant in code viewer - please align:
```suggestion
  template: `<kirby-stock-chart
  description="Accessibility description goes here"
  [options]="options"
  [data]="data"
  [height]="height">
</kirby-stock-chart>`,
```

File: apps/cookbook/src/app/examples/stock-chart-example/stock-chart-example.component.ts [hunk 27]
```diff
@@ -4,12 +4,22 @@ import { Options } from 'highcharts';
...
+};
+
+@Component(config)
```
Comment (jakobe) [inline apps/cookbook/src/app/examples/stock-chart-example/stock-chart-example.component.ts:20]: Doesn't compile - please align with existing examples:
```suggestion
@Component({
  selector: config.selector,
  template: config.template,
})
```

File: apps/cookbook/src/app/showcase/stock-chart-showcase/stock-chart-showcase.component.html [hunk 28]
```diff
@@ -1,7 +1,7 @@
 <div class="example">
```
Comment (jakobe) [inline apps/cookbook/src/app/showcase/stock-chart-showcase/stock-chart-showcase.component.html:1]: Why this wrapper??

File: apps/cookbook/src/app/showcase/stock-chart-showcase/stock-chart-showcase.component.html [hunk 29]
```diff
@@ -1,7 +1,7 @@
...
-  <cookbook-stock-chart-example></cookbook-stock-chart-example>
+  <cookbook-stock-chart-example #example></cookbook-stock-chart-example>
+  <cookbook-code-viewer [html]="example.template"></cookbook-code-viewer>
```
Comment (jakobe) [inline apps/cookbook/src/app/showcase/stock-chart-showcase/stock-chart-showcase.component.html:3]: Please align with existing showcases:
```suggestion
<cookbook-example-viewer [html]="example.template">
  <cookbook-stock-chart-example #example></cookbook-stock-chart-example>
</cookbook-example-viewer>
```

File: apps/cookbook/src/app/showcase/stock-chart-showcase/stock-chart-showcase.component.ts [hunk 30]
```diff
@@ -7,9 +7,6 @@ declare var require: any;
   styleUrls: ['./stock-chart-showcase.component.scss'],
 })
 export class StockChartShowcaseComponent implements OnInit {
-  exampleHtml: string = require('!raw-loader!../../examples/stock-chart-example/stock-chart-example.component.html')
-    .default;
-
   constructor() {}
 
   ngOnInit() {}
```
Comment (jakobe) [inline apps/cookbook/src/app/showcase/stock-chart-showcase/stock-chart-showcase.component.ts]: Delete empty methods:
```suggestion
```

File: apps/cookbook/src/app/examples/stock-chart-example/stock-chart-example.component.ts (hunk 26 above)
Comment (hawkish) [reply apps/cookbook/src/app/examples/stock-chart-example/stock-chart-example.component.ts:17]: Done.

File: apps/cookbook/src/app/examples/stock-chart-example/stock-chart-example.component.ts (hunk 27 above)
Comment (hawkish) [reply apps/cookbook/src/app/examples/stock-chart-example/stock-chart-example.component.ts:20]: Done.

File: apps/cookbook/src/app/showcase/stock-chart-showcase/stock-chart-showcase.component.html (hunk 28 above)
Comment (hawkish) [reply apps/cookbook/src/app/showcase/stock-chart-showcase/stock-chart-showcase.component.html:1]: Legacy I guess. I removed it.

File: apps/cookbook/src/app/showcase/stock-chart-showcase/stock-chart-showcase.component.html (hunk 29 above)
Comment (hawkish) [reply apps/cookbook/src/app/showcase/stock-chart-showcase/stock-chart-showcase.component.html:3]: Done.

File: apps/cookbook/src/app/showcase/stock-chart-showcase/stock-chart-showcase.component.ts (hunk 30 above)
Comment (hawkish) [reply apps/cookbook/src/app/showcase/stock-chart-showcase/stock-chart-showcase.component.ts]: Done.

**Pull request merged**